app/scripts/controllers/metametrics.test.js

```diff
@@ -604,6 +605,69 @@ describe('MetaMetricsController', function () {
     });
   });
 
+  describe('trackEvent', function () {
+    it('should call idempotent method createEventFragment, and finalizeEventFragment if actionId is provided', function () {
+      this.timeout(7000);
```

Comment:
What is this timeout for?

Reply:
I had added it to test async timeout: https://stackoverflow.com/questions/15971167/how-to-increase-timeout-for-a-single-test-case-in-mocha.

I removed it not as I updated the test case and it was no longer required.